drivers/bmp180/include/bmp180_internals.h

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (C) 2016 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    drivers_bmp180 BMP180
+ * @ingroup     drivers_sensors
+ * @brief       Internal addresses, registers, constants for the BMP180 sensor.
+ * @{
+ *
+ * @file
+ * @brief       Internal addresses, registers, constants for the BMP180 sensor.
+ *
+ * @author      Alexandre Abadie <alexandre.abadie@inria.fr>
+ */
+
+#ifndef BMP180_REGS_H_
+#define BMP180_REGS_H_
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name BMP180 I2C address
+ * @{
+ */
+#define BMP180_ADDR                   (0x77) // 7 bits address
```

Comment:
use c-style comment. And could you add the alignment of the 7-bit address (right as it seems)?! thx